TidepoolKit/Model/Datum/TCGMSettingsDatum.swift

```diff
@@ -0,0 +1,240 @@
+//
+//  TCGMSettingsDatum.swift
+//  TidepoolKit
+//
+//  Created by Darin Krauss on 11/1/19.
+//  Copyright © 2019 Tidepool Project. All rights reserved.
+//
+
+import Foundation
+
+public class TCGMSettingsDatum: TDatum, Decodable {
+    public typealias Units = TBloodGlucose.Units
+
+    public var manufacturers: [String]?
+    public var model: String?
+    public var serialNumber: String?
+    public var transmitterId: String?
```

Comment:
Not sure how granular the settings need to be, but do we want to include sensor information in these settings (use of a code or no code)?

How able the transmitter configuration (software version, firmware version, backfill supported, max run time, etc.)?

Reply:
We can definitely expand upon these when we update the Tidepool Data Model and API later.